app/views/shared/_sidebar_image.html.erb

```diff
@@ -0,0 +1,11 @@
+<% image ||= false %>
```

Comment:
I've removed this line. I think the problem this solves is if `image` isn't passed in as a local variable. Given this is a sidebar_image partial, that shouldn't ever happen.